doc/releases/changelog-0.27.0.md

```diff
@@ -592,7 +740,7 @@
 * Added a "Deprecations" page to the developer documentation.
   [(#3093)](https://github.com/PennyLaneAI/pennylane/pull/3093)
 
-* The example of the `FlipSign` template has been updated.
+* The example of the `qml.FlipSign` template has been updated.
   [(#3219)](https://github.com/PennyLaneAI/pennylane/pull/3219)
 
 <h3>Bug fixes</h3>
```

Comment:
lines 789 and 792 have the same GitHub link, can probably make them just one bullet point

Reply:
Sorry a little confused here, there's <700 lines in the changelog 🥴

Reply:
Thanks!